README.md

```diff
@@ -1,11 +1,20 @@
 # terraform-aws-mcaf-s3
 
+## Server access logging to the same bucket
+It's considered best practise to enable server access logging. That is why for compliance reasons we would like to have it enabled. However the default setting in this module is to log to the bucket itself. This may increase the costs as indicated by this message from AWS:
+
+>When your source bucket and target bucket are the same, additional logs are created for the logs that are written to the bucket. These extra logs can increase your storage billing and make it harder to find the logs that you're looking for.
+
+In short: we deemed compliance over costs.
+
+You can configure the bucket to send the logs another bucket by using the variable `logging`.
```

Comment:
```suggestion
## Server access logging

Server access logging provides detailed records for the requests that are made to a bucket and can useful in security and access audits. For this reason it is enabled by default and saves access logs to `s3_access_logs` in the same bucket. Be aware this can increase the costs as indicated by AWS:

> When your source bucket and target bucket are the same, additional logs are created for the logs that are written to the bucket. These extra logs can increase your storage billing and make it harder to find the logs that you're looking for.

This behaviour can be changed by modifying`var.logging`, either to log to another bucket or prefix, or disable server access logging by setting the value to `null`.
```

Reply:
Resovled in new commit